Examples/WildeGuess/WildeGuess/Components/InteractiveQuoteComponent.mm

```diff
@@ -52,19 +61,16 @@ + (instancetype)newWithQuote:(Quote *)quote
     }
     children:{
       {[CKOverlayLayoutComponent
-        newWithComponent:[QuoteComponent newWithQuote:quote context:context]
-        overlay:overlay]},
+        newWithComponent:[QuoteComponent newWithQuote:_quote context:_context]
+        overlay:(revealAnswer ? _overlay : nil)]},
```

Comment:
I think it's fairly problematic that we now **always** create `SuccessIndicatorComponent`, instead of only creating it when `revealAnswer` is `YES`.

Don't you think that's wasteful from a performance angle? Is there a solution?

Reply:
Yes, agree. We need to create it only when needed. See comment above.